EIPS/eip-bit_based_permission.md

```diff
@@ -34,24 +44,27 @@ _Note_ The following specifications use syntax from Solidity `0.8.7` (or above)
 - Permission MUST be defined as power of two
 - Permission MUST be unique
 
-An example of define new permissions and roles:
+### Defining new permissions and roles
 
 ```text
 const PERMISSION_NONE = 0;
 const PERMISSION_READ = 1;     // 2⁰
 const PERMISSION_WRITE = 2;    // 2¹
 const PERMISSION_EXECUTE = 4;  // 2²
 
-// Role admin = 4 | 2 | 1 = 7
-const ROLE_ADMIN = PERMISSION_READ | PERMISSION_WRITE | PERMISSION_EXECUTE;
 // Role operator = 1 | 2 = 3
 const ROLE_OPERATOR = PERMISSION_READ | PERMISSION_WRITE;
+
+// Role admin = 1 | 2 | 4 = 7
+const ROLE_ADMIN = PERMISSION_READ | PERMISSION_WRITE | PERMISSION_EXECUTE;
 ```
 
-An example of checking for required permissions:
+### Checking permission
 
+To verify if some permissions are owned, you need to apply a bitmask of it to the `ownedPermission`.
```

Comment:
> To verify if some permissions are owned, you need to apply a bitmask of it to the `ownedPermission`.

This could be rephrase to,
To verify if owned permission is a superset of required permission, you need to apply a bitmask of `requiredPermission` to the `ownedPermission`.